src/components/Disputes/DisputeInfo.js

```diff
@@ -25,6 +25,7 @@ import iconCourt from '../../assets/courtIcon.svg'
 import { addressesEqual } from '../../lib/web3-utils'
 import DisputeRoundPill from './DisputeRoundPill'
 import DisputeOutcomeText from './DisputeOutcomeText'
+import { Link } from 'react-router-dom'
```

Comment:
I think this should be imported from `@aragon/ui` ? 😄

Reply:
hahah yes , it was an auto import good catch